packages/jest-cli/src/reporters/utils.js

```diff
@@ -76,7 +76,7 @@ export const formatTestPath = (
   testPath: Path,
 ) => {
   const {dirname, basename} = relativePath(config, testPath);
-  return chalk.dim(dirname + path.sep) + chalk.bold(basename);
+  return slash(chalk.dim(dirname + path.sep) + chalk.bold(basename));
```

Comment:
Anybody with a Windows computer can confirm you can Ctrl+Click to get to test location? If it actually ever worked earlier of course

Reply:
didn't work from cmd at least. I didn't test in e.g. git bash or commandr (or what it's called, it's been years since I developed on windows)

Reply:
Looks like this should work with VSCode and terminals like git bash already use such paths. https://github.com/Microsoft/vscode/issues/27082